src/App.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "./App.css";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+} from "react-router-dom";
+import Home from "./components/Home.js";
+import Recipes from "./components/Recipes.js";
+import Recipe from "./components/Recipe.js";
+import RecipeForm from "./components/RecipeForm.js";
+
+function App() {
+  return (
+    <Router>
+      <div className="App">
```

Comment:
I think we can remove this one too

Reply:
I need to keep this one so the elements stay centered